.github/workflows/performance-test.yml

```
@@ -28,6 +28,7 @@ env:
 jobs:
   performance-job:
     name: performance
+    if: ${{ false }}
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)